Website: update Markdown blockquotes

eashaw opened the pull request:
Changes:
- Updated the custom blockquote renderer function in `to-html` to add `purpose="tip"` to blockquotes
- Added styles for new blockquotes to the handbook, docs, and articles stylesheets
   - If a `<blockquote>` HTML element is added with `purpose="quote"`, on fleetdm.com it will be rendered as:
    <img width="624" alt="image" src="https://user-images.githubusercontent.com/7445991/182730444-dcc192bc-761d-44d9-b7f6-4f6fedf1bb14.png">

   - If a  `<blockquote>` HTML element is added with `purpose="large-quote"`, on fleetdm.com it will be rendered as:
    <img width="633" alt="image" src="https://user-images.githubusercontent.com/7445991/182730575-3bbe96e1-a89b-40d9-8560-98dae5e6f97c.png">
- Added a blockquote section to the Markdown guide with examples of the different blockquotes styles

mikermcneil reviewed (changes requested):
> If the content of a Markdown blockquote is preceded by a `<!-- quote -->` HTML comment, the blockquote will be rendered as a `<blockquote>` element with purpose="quote"

Could the quote be instead written in markdown as:

```md
<blockquote purpose="large-quote">Break lamps and curl up into a ball…</blockquote>
```

```md
<blockquote purpose="quote">Break lamps and curl up into a ball…</blockquote>
```

That would require no changes to the build script.

eashaw commented:
@mikermcneil I added examples to the [Markdown guide](https://github.com/fleetdm/fleet/pull/7046/commits/b482b8483ca7dab14387e41e051a929b5e8532c8#diff-6a083f5d35839f082d36224a9ca9d3800abc9a81d5e1de3b975f50e97a8e0a73)

mikermcneil commented:
@eashaw 
<img width="702" alt="image" src="https://user-images.githubusercontent.com/618009/182904210-c1b906ce-f084-4107-aa5f-3f2176207c37.png">

eashaw commented:
@mikermcneil Yes, that would work! 

I've updated this PR to revert most of the changes to `to-html.js` and updated the examples in the Markdown guide.

mikermcneil reviewed (approved):
Nice work!
